PremiereClipboard/Program.cs

```diff
@@ -10,7 +10,7 @@ namespace PremiereClipboard
 {
     class Program
     {
-        static string trackFormat = "PProAE/Exchange/TrackItem";
+        const string trackFormat = "PProAE/Exchange/TrackItem";
```

Comment:
Constants should start with a capital letter.

source: https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions